app/components/mig/login/login.styl

```diff
@@ -1,3 +1,50 @@
+$mantis = #3cb9a0
+$carrotOrange = #ec932b
```

Comment:
Would break encapsulation of web components to have the `.styl` file import from a parent directory. I would instead look at this approach: https://github.com/DamonOehlman/stylify#adding-paths-for-require